site/styles/_nav.scss

```diff
@@ -397,7 +397,7 @@
     border: 0;
     border-radius: 50%;
     color: rgba(var(--body-color-rgb), 0.7);
-    height: 32px;
+    font-size: 24px;
```

Comment:
Looks like this gets overridden by the `.fa` class in font-awesome _core.scss.

Reply:
I'm not sure it does? When I remove this `font-size: 24px;` property the search close button gets smaller. Also I'm unsure where _core.scss is.

Reply:
_core.scss was in font-awesome... not that you've removed it, this issue is resolved!